### ISSUE TITLE
Organization settings tab is far below than shown

### ISSUE THREAD

**Issue by tashrifbillah** [MicrosoftDocs/azure-devops-docs#9730]
All right, back on Azure--please see how far below the `Organization Settings` tab is contrary to [the proximity](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/pools-queues?toc=%2Fazure%2Fdevops%2Forganizations%2Ftoc.json&bc=%2Fazure%2Fdevops%2Forganizations%2Fbreadcrumb%2Ftoc.json&view=azure-devops&tabs=yaml%2Cbrowser#what-is-a-maintenance-job) shown in the page. Took me a good while to realize.

![image](https://user-images.githubusercontent.com/35086881/100261819-50219c80-2f19-11eb-9e90-45c297efae5e.png)

It might be better to show such a long screenshot instead.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4f6f63ad-e95d-0105-c73a-de6e904edc09
* Version Independent ID: c97bfa1a-5aca-80a9-d9aa-8cca9987ad82
* Content: [Agents pools - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/pools-queues?toc=%2Fazure%2Fdevops%2Forganizations%2Ftoc.json&bc=%2Fazure%2Fdevops%2Forganizations%2Fbreadcrumb%2Ftoc.json&view=azure-devops&tabs=yaml%2Cbrowser)
* Content Source: [docs/pipelines/agents/pools-queues.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/master/docs/pipelines/agents/pools-queues.md)
* Product: **devops**
* Technology: **devops-cicd-agents**
* GitHub Login: @steved0x
* Microsoft Alias: **sdanie**

**Comment by steved0x** [comment]
#reassign steved0x

**Comment by steved0x** [comment]
@tashrifbillah Sorry that you ran into trouble - when I added that screenshot, I didn't want it to be super tall with all that whitespace so I condensed it down. I'll see if I can expand it or maybe add a cutout like this to indicate that it has been condensed.

![image](https://user-images.githubusercontent.com/1580889/102281237-922c7580-3efc-11eb-98e2-37030618b52b.png)

Sorry that you ran into trouble and I'm glad you got it figured out.
